### DISCUSSION TITLE
Is it possible to configure the Pull Request (PR) Author/Owner?

### DISCUSSION THREAD

**Discussion by nick-solly** [renovatebot/renovate#30679] [Request Help]
### What would you like help with?

Other

### How are you running Renovate?

None

### Please tell us more about your question or problem

- I'm using the `Mend Developer Platform` with the `Mend Renovate App for Bitbucket Cloud`
- I have configured Renovate to use a bot account as the commit author using `gitAuthor` in the json configuration file.
- However Renovate pull requests are being raised by a generic workspace account (`<workspace>@noreply.bitbucket.org`)
- When Bitbucket merges PR's it makes the author of the merge commit the PR raiser
- This means the merge commit is not authored by my bot account but instead this non-account that's associated to the workspace
- My Vercel pipelines then fail as the merge commit author is not an actual Bitbucket account and isn't registered in Vercel.

Is it possible to configure which user Renovate uses to raise the PR?

**Comment by rarkins** [comment]
Unfortunately this seems to be a Bitbucket limitation

**Reply by nick-solly** [reply]
Could `author` be added to the create PR request body here:

https://github.com/renovatebot/renovate/blob/20d3d484f6a2aab58a697a824b34878ab17a0608/lib/modules/platform/bitbucket/index.ts#L895

as per: https://developer.atlassian.com/cloud/bitbucket/rest/api-group-pullrequests/#api-repositories-workspace-repo-slug-pullrequests-post-request-body

**Reply by rarkins** [reply]
It could be worth trying although I'm not sure of how to test it using the CLI. @nabeelsaabna could you test this in the Mend dev platform to understand the impact?

**Reply by nabeelsaabna** [reply]
Do you want prefer adding a new Bitbucket only repository config for this ? 
(like we have https://docs.renovatebot.com/configuration-options/#bbusedefaultreviewers)

**Reply by rarkins** [reply]
It's possible we want to do this behavior whenever gitAuthor is set and we are running as an app?

**Reply by nick-solly** [reply]
Any feedback please @nabeelsaabna ?

**Reply by nabeelsaabna** [reply]
we need to convert this discussion to an issue with a bit more of details/ final solution in order to proceed

**Reply by nick-solly** [reply]
Great - @rarkins are you able to do that please?

**Reply by rarkins** [reply]
Not unless/until a customer asks for it

**Reply by nick-solly** [reply]
By customer I guess you mean paying customer? 😭

**Reply by rarkins** [reply]
Yes, see https://github.com/renovatebot/renovate/blob/main/CODE_OF_CONDUCT.md#how-we-prioritize-work

This feature request requires more details/research into APIs and Bitbucket apps. If you're not able to do it (for time or technical reasons) then that's ok, but it otherwise won't be prioritized by Mend employees unless there's a customer who considers it important.
